x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_form.tsx

```diff
@@ -136,8 +138,19 @@ const ConnectorFormComponent: React.FC<Props> = ({
   useEffect(() => {
     if (onChange) {
       onChange({ isValid: isFormValid, isSubmitted, isSubmitting, submit, preSubmitValidator });
+      if (isFormValid) {
+        resetErrorMessage?.();
```

Comment:
`isFormValid` is available "above" in the flyout(`/create_connector_flyout/index.tsx`) so it is not necessary to pass this function to the connector form. Why not keep all the logic there instead?